### PR TITLE
VZ 6901: 1.3.4 release notes

### PR DESCRIPTION
**Pull request by anders-swanson**

### CODE REVIEW

File: content/en/docs/releasenotes/_index.md [hunk 1]
```diff
@@ -4,6 +4,13 @@ linkTitle: Release Notes
 weight: 13
 draft: false
 ---
+### v1.3.4
+Fixes:
+
+- Updated Rancher to version v2.6.6.
+- Updated the Kiali image to fix CVEs.
```
Comment (kminder) [inline content/en/docs/releasenotes/_index.md:11]: Should "the" be removed to be consistent with previous "Updated Keycloak image to fix CVEs."?

Comment (rosemarymarano) [reply content/en/docs/releasenotes/_index.md:11]: The "the" should be in both sentences. Perhaps, he can fix that one too.

File: content/en/docs/releasenotes/_index.md [hunk 2]
```diff
@@ -4,6 +4,13 @@ linkTitle: Release Notes
 weight: 13
 draft: false
 ---
+### v1.3.4
+Fixes:
+
+- Updated Rancher to version v2.6.6.
```
Comment (kminder) [inline content/en/docs/releasenotes/_index.md:10]: Should "Updated Rancher to version v2.6.6." be under “Fixes” instead of “Component version updates:“?

Comment (MelwynPaul) [reply content/en/docs/releasenotes/_index.md:10]: Going by the recent Release Notes style (as in v1.3.0 section) - yes, we can add it under "Component version updates:" 

Update to-> 

Fixes:
- Updated the Kiali image to fix CVEs.
- Resolved an issue with Prometheus volume attachment during upgrade.

Component version updates:
- Rancher v2.6.6

**Review by MelwynPaul — APPROVED** [review]
Changes look good. 
Thank you.

**Pull request merged**